.github/workflows/pylint_check.yaml

```
@@ -42,9 +53,19 @@ jobs:
         shell: bash
         run: '[ -s "requirements.txt" ] && (python3 -m pip install --no-cache-dir -r requirements.txt) || (echo "No Python packages to install")'
         id: install-python-reqs
+#      - name: numpy install
+#        run: python3 -m pip install --upgrade --no-cache-dir numpy
+#      - name: enum34 install
+#        run: python3 -m pip install --upgrade --no-cache-dir enum34
+#      - name: piexif install
+#        run: python3 -m pip install --upgrade --no-cache-dir piexif
+#      - name: pygdal install
+#        run: python3 -m pip install --upgrade --no-cache-dir pygdal==2.4.2.*
+#      - name: agpypeline install
+#        run: python3 -m pip install --upgrade --no-cache-dir agpypeline==0.0.37
       - name: Run action pylint script
         shell: bash
-        run: '[ -s ".github/workflows/action_pylint.sh" ] && (chmod +x ".github/workflows/action_pylint.sh" && ./.github/workflows/action_pylint.sh) || (echo "Error running shell script")'
+        run: '[ -s ".github/workflows/action_pylint.sh" ] && (chmod +x ".github/workflows/action_pylint.sh" && ./.github/workflows/action_pylint.sh) || (echo "Not running special action script - empty or not found")'
```

```suggestion
         run: '[ -s ".github/workflows/action_pylint.sh" ] && (chmod +x ".github/workflows/action_pylint.sh" && ./.github/workflows/action_pylint.sh) || (echo "Error running shell script")'
```
The original text is actually correct